.github/workflows/publish.yaml

```diff
@@ -18,9 +18,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Install dependencies
-        run: pub get
+        run: dart pub get
```

Comment:
should all be `flutter` instead of `dart`

Reply:
https://docs.flutter.dev/development/packages-and-plugins/developing-packages#publish

Reply:
dart will work the same way, I believe they work the same way, believe it would be getting proxied

Reply:
flutter commands run the respective native build / install commands which dart by itself won't...

Reply:
also would be better to use the actual docker image compared to an action built and managed by an individual
https://github.com/subosito/flutter-action

Reply:
yeah, want to base it the other workflow for Unit Testing? it leverages that

Reply:
fixed :)